### PR TITLE
Add validation

### LINKED ISSUE #84: Add temperature validation for Tvib and Trot

### PR DESCRIPTION
**Pull request by arunavabasu-03**
Hey @suzil, I do the validation thing that doesn't allow negative values, please check 😀 .
Fixes #84 .

### CODE REVIEW

File: website/src/components/CalcSpectrum.tsx [hunk 1]
```diff
@@ -126,17 +126,32 @@ export const CalcSpectrum: React.FC = () => {
...
       if (!params.tvib) {
         updatedValidationErrors.tvib =
           "Tvib must be defined when running non-equilibrium calculations";
       }
+      // else if (params.tvib < 1 || params.tvib > 9000) {
+      //   updatedValidationErrors.trot = "Tvib must be between 1K and 9000K";
+      // }
+      else {
+        updatedValidationErrors.tvib = undefined;
+      }
```
Comment (suzil) [inline website/src/components/CalcSpectrum.tsx:154]: @arunavabasu-03 Are these changes intended to be checked in? As it seems like this is removing existing validation for trot & tvib?

Comment (arunavabasu-03) [reply website/src/components/CalcSpectrum.tsx:154]: @suzil, I check that.

**Review by suzil — APPROVED** [review]
